main/integration-tests/__test__/api-tests/studies/study-upload-requests.test.js

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
What do you think of naming the file get-study-upload-requests.test.js, so that we are consistent with the standard RESTful operations?